new build a self-signed jwt guide (#411)
okta/okta-developer-docs: packages/@okta/vuepress-site/guides/build-self-signed-jwt/jwt-with-private-key/index.md

---
title: Build a JWT With a Private Key
---

> If you configured your client to use the `private_key_jwt` client authentication method, you must provide the `client_id` in a JWT that you sign with your private key using an RSA or ECDSA algorithm (RS256, RS384, RS512, ES256, ES384, ES512). 

Reviewer: ```suggestion
If you configured your client to use the `private_key_jwt` client authentication method, then you want to build a JWT that you sign with your private key using an RSA or ECDSA algorithm (RS256, RS384, RS512, ES256, ES384, ES512). 
```

Reviewer: `client_id` is required as `sub`  and `iss` in all cases.
